# MouseMsgPassIRL.msg

# float32[] xyz

geometry_msgs/Pose[] xyz